package.json

```diff
@@ -56,7 +56,10 @@
     "typescript": "3.8.3"
   },
   "dependencies": {
+    "@types/axios": "^0.14.0",
     "@types/cheerio": "0.22.16",
+    "@types/circular-json": "^0.4.0",
```

Comment:
please change all versions to not include a caret

Reply:
Fixed.